source/index.html.md.erb

```diff
@@ -3222,13 +3249,17 @@ PagedResult<Column> columns = smartsheet.sheetResources().columnResources().list
 ```
 
 ```python
-action = smartsheet.Sheets.get_columns(sheetId, include_all=True)
-columns = action.data
+# Sample 1: List all
+response = ss_client.Sheets.get_columns(
+    9283173393803140,       # sheet_id
+    include_all=True)
+columns = response.data
 
-# Paginate the list of columns (100 columns per page)
-action = smartsheet.Sheets.get_columns(sheetId)
-pages = action.total_pages
-columns = action.data
+# Sample 2: Paginate the list (100 columns per page)
```

Comment:
This example doesn't specify the paging params. Should either be explicit, or simply omit. (Column list is unlikely to be too large to get in one gulp)